tests/.env

```diff
@@ -0,0 +1,9 @@
+COGNITO_DOMAIN = "https://maap-stac-auth-dev.auth.us-west-2.amazoncognito.com"
+SCOPE = "MAAP-STAC-ingestion-registry-server/stac:register"
+STAGE = "dev"
+STAC_REGISTER_SERVICE_ID = "MAAP-workflows"
+BASE_URL = "https://wynt4rero2.execute-api.us-west-2.amazonaws.com/test"
```

Comment:
oh right, this is the ingestor endpoint and not the stac-api endpoint. This and the cognito_domain should at least be moved into github secrets when we get permissions

Reply:
I agree.